Internal/Deploy.ps1

```diff
@@ -162,12 +150,13 @@ try {
         Write-Host -ForegroundColor Yellow "Deploying to $repo"
 
         try {
-            $serverUrl = "https://$($config.githubOwner):$token@github.com/$($config.githubOwner)/$repo.git"
+            $serverUrl = "https://$($config.githubOwner)@github.com/$($config.githubOwner)/$repo.git"
             if (Test-Path $repo) {
                 Remove-Item $repo -Recurse -Force
             }
             invoke-git clone --quiet $serverUrl
             Set-Location $repo
+            invoke-gh auth setup-git

```

Comment:
@aholstrup1 I tested this out today, to get rid of my OrgPAT.

But I believe that `invoke-gh auth setup-git` should be moved a couple of lines up, above `invoke-git clone`.
One can even argue for having this line above the try block, since we need this authentication both in the try and catch block :)